.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "bliss"]
```

Comment:
Why is this removed? I thought this was required for pangolin.

Reply:
It is already vendored in external directly rather than as a submodule. The submodule itself was just something @chenxuhao made to track bliss upstream but it seems simpler to avoid inserting an intermediary.

Reply:
Yes, it's better it's not a submodule but that is not related to this PR. As long as pangolin builds don't break, it's fine to be included in this PR.

Reply:
FWIW, the reason the commit is here is because we have to enable submodules for GPU builds, so I took the opportunity to clean up our submodule configuration, which has these unused entries.

Reply:
Ok, makes sense.